### DISCUSSION TITLE
Adjusting the gain from the ais-catcher's web page.

### DISCUSSION THREAD

**Discussion by Taxom** [jvde-github/AIS-catcher#71] [Ideas]
Is it possible to add reinforcement adjustment and other settings of the receiver on the web of the AIS trap? Of course with authentication or on other port.
Sorry for the flow is possible stupid ideas.
ps: You carry them out faster than I have time to figure out their functionality. I am now studying Markdown to use the previous function.

With great respect.

Max

**Comment by jvde-github** [comment]
Like that. The program was originally not designed to enable this easily (was not even planning the web client) but have been slowly moving in this direction with the JSON config and some other refactoring. Still quite some legacy changes to rework to make this possible. It is a great dot on the horizon and would definitely like this feature myself as well.  Keep you posted.

**Comment by g7ruh** [comment]
Hi Max

I have been using AIS-catcher plots of Frequency Shift and Signal Level to help me fine-tune the RTL Dongle I am using.

I will post a new discussion here to explain what I did, later this week as it may trigger ideas about what can be done. I was quite impressed with the end result.

This comment is just to let you know about it, and that further info is on the way.

Roger

**Comment by Taxom** [comment]
I wanted to add a simple, in my opinion, option to delimit access to settings from the web interface. Just making access to the interface from two different ports, one with the possibility of setting the second is not. Then the one that does not have access to the settings can be opened through the "port  forwarding" for the outside world.

**Reply by jvde-github** [reply]
Great idea. In principle, you can already start multiple webservers with differing settings, e.g.
```
AIS-catcher -x 192.168.1.120 4002 -N 8100 message on -N 8101 message off -N 8102 lat 3.1 lon 40.4
```
The main problem is I think finding the time and energy to allow for settings to be tweaked in the web-interface. Nice idea, but not the highest priority unfortunately.
